release-notes/3.1/3.1-known-issues.md

```diff
@@ -18,6 +18,26 @@ We are working to update our installers for the new Catalina requirements. These
 
 ## ASP.NET Core
 
+### 3.1.0
+
+* **The 3.1.0 Hosting Bundle for Windows installs a .NET Core Runtime incorrectly branded "3.1.0 Preview 3".**
+
+This is a cosmetic bug: the hosting bundle is expected to function normally.
+
+This will be fixed with 3.1.1. More information at [dotnet/core#3962](https://github.com/dotnet/core/issues/3962).
+
+The Hosting Bundle briefly shows that it is installing "Microsoft .NET Core Runtime - 3.1.0 Preview 3". That string is also visible in Add/Remove Programs after the install completes. This is **not** the old November 2019 3.1.0 Preview 3 Runtime. The Hosting Bundle is installing a more recent build of 3.1.0 that has incorrect branding.
+
+* **If the 3.1.0 SDK and Hosting Bundle were previously installed on Windows, installing the 3.1.0 .NET Core Runtime directly may remove important files such as `hostpolicy.dll`.**
+
+To fix this, repair the .NET Core Runtime installation. (Execute the installer again and click Repair, or find the .NET Core Runtime 3.1.0 installation in Add/Remove Programs, select Modify, then click Repair.)
+
+This will be fixed with 3.1.1. More information at [dotnet/core#3962](https://github.com/dotnet/core/issues/3962) and [dotnet/runtime#703](https://github.com/dotnet/runtime/issues/703).
```

Comment:
At this point, are we sure this will make 3.1.1, or might it slip to 3.1.2?  @mmitche

Reply:
The infra PRs aren't necessary to prevent it from happening again, we (build + release drivers) just need to learn from the 3.1.0 issues and be more careful. The PRs just make the system more robust so we don't need to be as careful.